### ISSUE TITLE
How lo use same structure for different pages.

### ISSUE THREAD

**Issue by gonzalovilaseca** [backbee/BackBee#569]
So lets say we want to have a Twitter block on the right column of all pages of our website, is there a way to do it other that adding it every time we create new content?
Thanks.

**Comment by mickaelandrieu** [comment]
Hi @gonzalovilaseca,

yes. The best you can do is create a new Layout and define a `defaultClassContent` in the aside Column.

It's a little bit documented here => http://docs.backbee.com/developer-documentation/cookbooks/layout_creation/

Regards

**Comment by crouillon** [comment]
Hi @gonzalovilaseca,

On the same idea of @mickaelandrieu, you can also define the right coumn of your layout as inherited.
Doing so the right column of a page (the root one for example) will be shared by every descendant pages : droping a twitter bloc once on the rigth column of the page will display it on every pages.

Regards

**Comment by gonzalovilaseca** [comment]
great, thanks to both for your help.

**Comment by gonzalovilaseca** [comment]
Back to this question: can I override the block of the parent in just one of the child pages?

So I want most of the chils pages to inherit from the parent, but on one of them I want to replace the block. Is that possible?

Thxs.

**Comment by mickaelandrieu** [comment]
@gonzalovilaseca yes, sure.  You can remove any block from pages with the toolbar in "blocks"  context.

What is your need?

**Comment by gonzalovilaseca** [comment]
Say we have a `blog` layout, with main and left column. 
Pages that inherit from this layout should display the `blog` layout, unless we add some specific blocks to any child page.

**Comment by mickaelandrieu** [comment]
hmm... sound complicated to do vs make a new layout in my opinion /c @crouillon @eric-chau .
Using a listener on class content pre render event (http://docs.backbee.com/developer-documentation/components/event_listener/#events-from-class-content-component), I think you can set a new Layout dynamicaly for this page.

Interesting issue, thank you for your interest @gonzalovilaseca !

**Comment by gonzalovilaseca** [comment]
@mickaelandrieu don't worry, we'll follow the template approach, thanks!
